apps/website/app/controllers/orders_controller.ts

```diff
@@ -112,7 +69,7 @@ export default class OrdersController {
       description = `Payment for order: ${description.slice(0, -2)}`;
 
       // Create the order and associated products
-      const order = await Order.create({ userId, name, nif, address });
+      const order = await Order.create({ userId: authUser.id, name, nif, address });
```

Comment:
Shouldn't this also move to a service function?

Reply:
you're right - will fix